Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+#!groovy
+
+node {
+ stage 'Checkout'
```

Comment:
"Checkout" isn't maybe the best choice here - maybe "Build"? Is there a standard naming convention for pipeline stages proposed by Jenkins?